src/android/LocalNotifications.java

```diff
@@ -86,10 +87,20 @@ private void showNotification(JSONArray args) throws JSONException {
         int requestCode = new Random().nextInt();
         PendingIntent contentIntent = PendingIntent.getActivity(context, requestCode, notificationIntent, PendingIntent.FLAG_UPDATE_CURRENT);
 
+        NotificationManager mNotificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+
+        NotificationCompat.Builder mBuilder = null;
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
+            int importance = NotificationManager.IMPORTANCE_DEFAULT;
+            NotificationChannel notificationChannel = new NotificationChannel("ID", "Name", importance);
```

Comment:
The channel should be named differently. Maybe id="default" and name="Default"? Also, just use `NotificationManager.IMPORTANCE_DEFAULT` directly here rather than creating the `importance` variable.